### DISCUSSION TITLE
select 组件是否支持 再次打开时不保留上一次的滚动位置?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#43258] [Q&A]
如题

**Comment by zombieJ** [comment]
这个目的是什么？如果不用保留可以用 open 受控 scrollTo 到顶端试试

**Comment by vaynevayne** [comment]
scrollTo 是哪里的 api, 文档里没看到,具体怎么用?

我的需求是这样的, 不太想加更多的 open 状态, 能否直接调用 select ref 上的某个 api来解决? 
```jsx
onDropdownVisibleChange ={ visible => {
if(visible) selectRef.current. scrollTo() // 
}}
```

**Comment by linxianxi** [comment]
需求有点怪，滚动到选的位置体验才是比较好的。要做只能这样，但是会闪一下，有延迟。
<img width="668" alt="image" src="https://github.com/ant-design/ant-design/assets/47104575/857eb1ae-1a52-47b9-a0f8-d1c0a05c9a6e">

**Comment by vaynevayne** [comment]
因为重新排序过列表: 有上千的选项, 为了方便,展开时,把选中的选项放在前面, 所以再次展开时,直接显示第一个元素就行

在 issue 里找到一个有用的方法, 但不知道能用多久, 毕竟文档没写
```jsx
selectRef.current.scrollTo(0)
```
